libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -55,6 +71,53 @@ export class SkyInputBoxComponent implements OnInit {
   @Input()
   public disabled: boolean | undefined;
 
+  /**
+   * The text to display as the input's label and in known validation error messages. The label
+   * will automatically be associated with the `input`, `select`, or `textarea` element included
+   * in the input box.
+   */
+  @Input()
+  public labelText: string | undefined;
+
+  /**
+   * The maximum number of characters allowed in the input. A [SKY UX character count](https://developer.blackbaud.com/skyux/components/character-count)
+   * will be placed on the input element with the appropriate validator.
```

Comment:
This also displays an error message if they go over, right? Or just prevents them from typing more? Also uh is it working

Reply:
I'm relying on the hyperlink to the character count component to describe all of its functionality.